test/spec/modules/ssp_genieeBidAdapter_spec.js

```diff
@@ -358,6 +358,28 @@ describe('ssp_genieeBidAdapter', function () {
         expect(request[0].data.extuid).to.deep.equal(`im:${imuid}`);
       });
 
```

Comment:
[SHOULD].  
既存のimuidのテストに習ってrequest[0].data.extuidからid5を確認するテストも書いた方が良いですね。

以下が参考になるかと思います。   
本リポジトリはprebid-analyticsと違い、`spec.buildRequests`でエンコードが走らないっぽいので少し変えています。

```suggestion
      it('should include only id5id in extuid query when only id5id exists', function () {
        const id5id = 'id5id';
        const request = spec.buildRequests([{...BANNER_BID, userId: {id5id: {uid: id5id}}}],);
        expect(request[0].data.extuid).to.deep.equal(`id5:${id5id}`);
      });

      it('should include id5id and imuid in extuid query when id5id and imuid exists', function () {
        const imuid = 'b.a4ad1d3eeb51e600';
        const id5id = 'id5id';
        const request = spec.buildRequests([{...BANNER_BID, userId: {id5id: {uid: id5id}, imuid: imuid}}]);
        expect(request[0].data.extuid).to.deep.equal(`id5:${id5id}\tim:${imuid}`);
      });

      it('should not include the extuid query when both id5 and imuid are missing', function () {
        const request = spec.buildRequests([BANNER_BID]);
        expect(request[0].data).to.not.have.property('extuid');
      });
```